### ISSUE TITLE
appsody build --buildah looks for docker CLI

### ISSUE THREAD

**Issue by nastacio** [appsody/appsody#909]
**Describe the bug**
As context, I was trying to build an appsody project inside https://learn.openshift.com/playgrounds/openshift42/

It looks like `appsody build --buildah` attempted to execute the `docker image ls ...` command, but I believe the --buildah option should  preclude invocations of the docker CLI. Maybe it is just a bad message after an unexpected `buildah` failure (see below) , but the combination of error and ensuing message causes the impression that the `--buildah` option does not prevent appsody from depending on the `docker` CLI.

**To Reproduce**
1. Launch a playground at https://learn.openshift.com/playgrounds/openshift42/
2. Type the following commands
```
curl -sL https://github.com/appsody/appsody/releases/download/0.5.8/appsody-0.5.8-1.x86_64.rpm -o appsody.rpm
curl -sL https://cbs.centos.org/kojifiles/packages/libseccomp/2.4.1/0.el7/x86_64/libseccomp-2.4.1-0.el7.x86_64.rpm -o libseccomp.rpm
yum install -y ./appsody.rpm ./libseccomp.rpm buildah 

git clone https://github.com/nastacio/appsody-nodejs
cd appsody-nodejs
appsody build --buildah
```

**Expected behavior**
I expected it to work, but it failed due to a problem with buildah (bad, but not important) .

**Actual behaviour**
The ensuing error message indicates that `appsody` attempted to invoke `docker` in the error path:

```
Your Appsody project name has been set to appsody-nodejs
Extracting project from development environment
Pulling docker image docker.io/kabanero/nodejs-express:0.2
Running command: buildah pull docker.io/kabanero/nodejs-express:0.2
Error during unshare(CLONE_NEWUSER): Operation not permitted
level=error msg="error parsing PID \"\": strconv.Atoi: parsing \"\": invalid syntax"
level=error msg="(unable to determine exit status)"
[Warning] Docker image pull failed: exit status 1
[Warning] Could not run docker image ls -q for the image: kabanero/nodejs-express:0.2 error: exec: "docker": executable file not found in$PATH Check to make sure docker is available.
[Error] Could not find the image either in docker hub or locally: docker.io/kabanero/nodejs-express:0.2
```

**Environment Details (please complete the following information):**
 - OpenShift 4.2 Playground

**If applicable please specify:**
 - CLI version: appsody 0.5.8
 - Stack you are using:
nodejs-express, but not important, happens regardless of stack, as long as stack uses image from public docker registry.

**Screenshots**

**Additional context**
Seems related, or a duplicate of #880

**Comment by kylegc** [comment]
I think this will be fixed by https://github.com/appsody/appsody/issues/880

**Comment by upLukeWeston** [comment]
Fixed by #911 .
